build.gradle

```diff
@@ -1,21 +1,16 @@
 import com.github.jengelman.gradle.plugins.shadow.ShadowPlugin
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
 
 plugins {
     id 'groovy'
     id 'project-report'
     id 'idea'
     id 'java-gradle-plugin'
-    // id 'signing'
```

Comment:
I think I will re-add `signing` as `plugin-publish` supports it natively. I had turned this off when troubleshooting errors with publishing the 8.1.0 plugin.

Reply:
I'll put it back, believe you just set `signing.required = false` will disable the sign tasks.

Reply:
https://github.com/johnrengelman/shadow/pull/831#discussion_r1119028819